MantidQt/MantidWidgets/src/InstrumentView/InstrumentWidgetPickTab.cpp

```diff
@@ -1441,9 +1418,8 @@ void DetectorPlotController::prepareDataForSumsPlot(int detid,
     }
   }
 
-  if (err) {
+  if (err)
     std::transform(err->begin(), err->end(), err->begin(), Sqrt());
-  }
```

Comment:
There is `operator+` and `+=` for `Histogram` (`HistogramMath.h`), that will do the error propagation internally and you can remove a lot of code above.